tests/models/test_openai.py

```diff
@@ -804,6 +804,23 @@ async def test_image_url_input(allow_model_requests: None):
     )
 
 
+async def test_image_url_input_force_download(allow_model_requests: None, openai_api_key: str):
+    provider = OpenAIProvider(api_key=openai_api_key)
+    m = OpenAIChatModel('gpt-4.1-nano', provider=provider)
+    agent = Agent(m)
+
+    result = await agent.run(
+        [
+            'What is this vegetable?',
+            ImageUrl(
+                force_download=True,
+                url='https://t3.ftcdn.net/jpg/00/85/79/92/360_F_85799278_0BBGV9OAdQDTLnKwAPBCcg1J7QtiieJY.jpg',
+            ),
+        ]
+    )
+    assert result.output == snapshot('This vegetable is a potato.')
```

Comment:
I think you'll need to regenerate the cassette, try `uv run pytest tests/models/test_openai.py::test_image_url_input_force_download --record-mode=rewrite`